avocado_virt/plugins/virt_bootstrap.py

```diff
@@ -75,10 +75,10 @@ def run(self, args):
             actual_sha1 = crypto.hash_file(filename=jeos_dst_path,
                                            algorithm="sha1")
         else:
-            actual_sha1 = '0'
+            actual_sha1 = 'FILE DOES NOT EXISTS LOCALLY'
```

Comment:
I don't get it... you mean human friendly, but this is not used in the UI, right?

Reply:
Anyway, this should read (in case you choose to go with this approach): `FILE DOES NOT EXIST LOCALLY`